doc/performance_blog_post/performance_blog_post.md

```diff
@@ -0,0 +1,109 @@
+
+In order to make the tests more reliable we run each test 5 times. Then we drop the first 2 results, since they are usually slower, since S3 will keep the files in cache for the following runs. Then we build the average of the remaining 3 runs. We decided for the average since it will again result in a value that customers can expect.
+
+### Test Fixtures
+
+For our performance test we need some test fixtures. For example one big parquet file on S3 that we load with a Virtual Schema and measure the performance.
+
+The simplest option for this is to create this file with some tool locally and store it forever on S3. This approach has the advantage, that its easy to create. On the other hand, it's hard to change, and it's also not apparent from the source code how the test data is created. Since out projects are open source, its also important that all (non Exasol-) contributors can run the tests. So we would need to make our S3 bucket publicly available which could cause costs.
+
+These issues can be solved by creating the test data on-the-fly before running the test. Then the test-data is documented and under version control. That can be very helpful, to make sure that there were no changes on the test fixture between two runs.
+
+One downside of this approach is that it only works for rather small test files. For larger files, creating and uploading the files gets too expensive and slow. To solve this we added a cache: Before uploading the test files, we crate a checksum and check if the files are already in place. Only if something was changed we re-upload the files.
+
+## Displaying Performance Tests Results
+
+In order to evaluate the test results, and discuss them, we need to display them somehow. For that we decided for a [Redash](https://redash.io/) dashboard. Redash is an open source tool that allows you to create online dashboards for SQL queries. So we push our performance test results into an Exasol database and use Redash to evaluate them.
+
+## Setting Goals
+
+The first test run generated the following results:
+
+| Test Case                          | Time in s |
+|------------------------------------|-----------|
+| testLoadManyParquetRows            | 23        |
+| testLoadLargeParquetRows           | 13.02     |
+| testLoadManyParquetColumns         | 12.9      |
+| testLoadManyParquetRowsFromOneFile | 74.64     |
+| testManySmallJsonFiles             | 685.87    |
+| testLoadFewBigSalesParquetFiles    | 662.28    |
+| testLoadSalesParquetFiles          | 219.21    |
+
+The first 4 cases (`testLoadManyParquetRows`, `testLoadLargeParquetRows`, `testLoadManyParquetColumns` `testLoadManyParquetRowsFromOneFile`) all load the same amount of data. They just differ on how the data is distributed over rows, files and columns.
```

Comment:
It is not very clear how are they same amount of data. But maybe you meant same amount of rows?

If it is same data (for example, they are all 2GB data), what is the difference between first two files and the one with many columns?